target/Target/AntiPattern.hs

```diff
@@ -92,3 +92,11 @@ stanSlashesWindows' = "xxx" </> "asd\\asd"
 
 stanSlashesNo :: FilePath
 stanSlashesNo = "xxx" </> "asd"
+
+stanStringsAreBad :: String
+stanStringsAreBad = "please never use this data type"
+
+stanFunctionReturningString :: a -> String
+stanFunctionReturningString = const "nooo"
+
```

Comment:
Let's also add another test, like

```haskell
stanFunctionTakingString :: String -> Bool
stanFunctionTakingString = const True
```

to make sure that it won't be triggered.

Reply:
Why it should *not* be triggered if it is a function argument? 🤔